linux/runtime.go

```diff
@@ -65,6 +65,7 @@ func init() {
 
 var _ = (runtime.Runtime)(&Runtime{})
 
+// Config options for the runtime
```

Comment:
I usually try to adhere to the `Name <verbs> <desc>` for doc. Not sure if it's the actually Golang convention. I have doubt now that I see your added godoc in this PR :sweat_smile: